pydicom/compat.py

```diff
@@ -1,34 +1,21 @@
 # Copyright 2008-2018 pydicom authors. See LICENSE file for details.
-"""Compatibility functions for python 2 vs later versions"""
+"""Compatibility functions for previous Python 2 support"""
 
 # These are largely modeled on Armin Ronacher's porting advice
 # at http://lucumr.pocoo.org/2013/5/21/porting-to-python-3-redux/
 
 import sys
 
-in_py2 = sys.version_info[0] == 2
 in_PyPy = 'PyPy' in sys.version
 
 # Text types
 # In py3+, the native text type ('str') is unicode
-# In py2, str can be either bytes or text.
-if in_py2:
-    text_type = unicode
-    string_types = (str, unicode)
-    char_types = (str, unicode)
-    number_types = (int, long)
-    int_type = long
-else:
-    text_type = str
-    string_types = (str, )
-    char_types = (str, bytes)
-    number_types = (int, )
-    int_type = int
+text_type = str
+string_types = (str, )
+char_types = (str, bytes)
+number_types = (int, )
+int_type = int
```

Comment:
Probably we can remove some of these - except maybe for `char_types` they can just be replaced by the actual type.